### ISSUE TITLE
Add a script contribution.

### ISSUE THREAD

**Issue by araujo-sena-daniel** [cejug/hurraa#140]
Add a script contribution to help people who want to contribute to the project.

**Comment by helio-frota** [comment]
https://github.com/cejug/hurraa/blob/master/configWildfly8.sh
1) Instalar o mysql/mariadb, criar o banco hurraa no mysql/mariadb usualmente
2) Rodar o script como root e informar o usuário e senha do banco.
3) Apontar o WTP do eclipse para o wildfly que ficará em /opt/

lista:
https://groups.google.com/forum/#!msg/cejug/f4A58p-zrL8/3_C3oNcNtRQJ

**Comment by paulojribp** [comment]
@eprogramming please keep the default contribution language :)

I believe that this script is not tech one like you quoted. This is a script step by step like you have made to Yougi. This three steps you have cited is so simple and have no details about how to do it in Eclipse or how to compile with maven. I believe this is why @araujo-sena-daniel open this issue.

@araujo-sena-daniel, if I was right, you can reopen it and contribute with this script. Otherwise, we can keep this closed.

**Comment by helio-frota** [comment]
This is a script step by step like you have made to Yougi

@paulojribp , please read. This script was made by Clairton and this is for Hurraa.

**Comment by araujo-sena-daniel** [comment]
@paulojribp ,
You're right. The script is actually very simple. I needed more details about the project configuration. It would be a step by step tutorial. The aim is to help people who have more basic knowledge. Sorry my current lack of knowledge, but do not know how to reopen the task, and I still have no ability to do this script.
Thank you!

**Comment by emanuelmm** [comment]
@araujo-sena-daniel, @paulojribp and @eprogramming 
Friends how helio said, isn't this the same that clairton did? 
https://github.com/cejug/hurraa/blob/master/configWildfly8.sh

**Comment by efraimgentil** [comment]
This script is not a executable or a way to fast install the Hurraa, is more like a tutorial step by step of what the new contributors would need to do to contribute, the pratices ,configurations, instructions  etc.

The script made by Clairton is a fast way to install the Hurraa and run the application, not a script (steps)  to contribute with the Hurraa

**Comment by helio-frota** [comment]
Ok now it is clear.
Diff between step-by-step (script) commands and automatic (shell) script.

**Comment by paulojribp** [comment]
Thats right @efraimgentil. Thanks.
